### DISCUSSION TITLE
Get much more traction & contribution in the project through GSoC-24

### DISCUSSION THREAD

**Discussion by zeekhuge** [outline/outline#5892] [General] [answered]
Hi @tommoor 

You have put amazing effort into building this repository. 
Any plans to bring in the organization under [GSoC](https://summerofcode.withgoogle.com/) ?

I have been a student as well as mentor (for 3 years) at Beagleboard.org for GSoC & its a pretty great place to get more traction as well as contributions into the project.

If you'd agree, I'd be up to help you with all the steps, and can help in mentoring the students as well for the program.
GSoC-24 will probably be announced next month & the Org application would probably start around Dec/Jan. 

Let me know if you have any plans/thoughts on this. Also, I tried to (very) quickly search about GSoC with BSL license & could not find much info on the same.

Let me know if you'd be open to the idea of participating in the Program and we can then look more into it.

Again, thanks for Outline,

**Comment by tommoor** [comment, marked as answer]
Hey, I'm afraid projects must use an [OSI-approved license](https://developers.google.com/open-source/gsoc/faq#what_are_the_eligibility_requirements_for_a_mentoring_organization) – BSL does not fall into this category.
